UserSync/Web.config

```diff
@@ -186,10 +186,6 @@
 				<assemblyIdentity name="System.Net.NetworkInformation" publicKeyToken="B03F5F7F11D50A3A" culture="neutral"/>
 				<bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0"/>
 			</dependentAssembly>
-			<dependentAssembly>
```

Comment:
This file was not found at runtime. Removed it, and the sample works fine with it.